src/Command/ListCommand.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Lolart\EzShell\Command;
+
+use eZ\Publish\API\Repository\Values\Content\LocationQuery;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
+use eZ\Publish\API\Repository\Values\Content\Query\SortClause\ContentName;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ListCommand extends Command
+{
+    protected function configure()
+    {
+        $this
+            ->setName('ez:ls')
+            ->setDescription('List the content at a location.')
+            ->setDefinition([
+                new InputArgument('source-id', InputArgument::REQUIRED, 'Location ID of the source')
+            ]);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        /** @var $repository \eZ\Publish\API\Repository\Repository */
+        $repository = $this->getScopeVariable('repository');
+        $searchService = $repository->getSearchService();
+
+        $query = new LocationQuery();
+        $query->filter = new Criterion\LogicalAnd(
+            [
+                new Criterion\ParentLocationId($input->getArgument('source-id')),
+            ]
+        );
+        $query->limit = 0;
+        $query->sortClauses = [new ContentName()];
+
+        $searchResult = $searchService->findLocations($query);
+
+        $nbResults = $searchResult->totalCount;
+
+        $pageSize = 100;
+        $query->limit = $pageSize;
+
+        while ($searchHits = $searchService->findLocations($query)->searchHits) {
+            foreach ($searchHits as $searchHit) {
+                $output->writeln($searchHit->valueObject->id . " " . $searchHit->valueObject->contentInfo->name);
```

Comment:
If this command is only going to be used in PsySH (i.e. not shared with other console apps) you can paginate output.

* Paginate a single string, [like the `parse` command](https://github.com/bobthecow/psysh/blob/master/src/Psy/Command/ParseCommand.php#L120).
* Paginate a closure, [like the `doc` command](https://github.com/bobthecow/psysh/blob/master/src/Psy/Command/DocCommand.php#L70-L81).
* Start/stop pagination, [like the `ls` command](https://github.com/bobthecow/psysh/blob/master/src/Psy/Command/ListCommand.php#L129-L140).